apis/logging/v1/output_types.go

```diff
@@ -256,3 +259,12 @@ type GoogleCloudLogging struct {
 	//LogID is the log ID to which to publish logs. This identifies log stream.
 	LogID string `json:"logId,omitempty"`
 }
+
+// Splunk Deliver log data to Splunk’s HTTP Event Collector
+// Provides optional extra properties for `type: splunk_hec` ('splunk_hec_logs' after Vector 0.23
+type Splunk struct {
+	// Fields to be added to Splunk index. https://docs.splunk.com/Documentation/Splunk/8.0.0/Data/IFXandHEC
+	// Should be a valid JSON string
```

Comment:
```suggestion
	// Should be a valid JSON object
```
I think "JSON sting" is confusing here, it's a string representation of a JSON object value not a JSON string value.